#### On which genes do we see changes because of dexamethasone

In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

In [9]:
counts = pd.read_csv("https://bioboot.github.io/bimm143_S20/class-material/airway_scaledcounts.csv", index_col="ensgene")
metadata = pd.read_csv("https://bioboot.github.io/bimm143_S20/class-material/airway_metadata.csv", index_col="id")

In [11]:
counts.shape

(38694, 8)

In [12]:
counts.head()

,SRR1039508,SRR1039509,SRR1039512,SRR1039513,SRR1039516,SRR1039517,SRR1039520,SRR1039521
ensgene,,,,,,,,
ENSG00000000003,723.0,486.0,904.0,445.0,1170.0,1097.0,806.0,604.0
ENSG00000000005,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
ENSG00000000419,467.0,523.0,616.0,371.0,582.0,781.0,417.0,509.0
ENSG00000000457,347.0,258.0,364.0,237.0,318.0,447.0,330.0,324.0
ENSG00000000460,96.0,81.0,73.0,66.0,118.0,94.0,102.0,74.0


In [13]:
metadata

,dex,celltype,geo_id
id,,,
SRR1039508,control,N61311,GSM1275862
SRR1039509,treated,N61311,GSM1275863
SRR1039512,control,N052611,GSM1275866
SRR1039513,treated,N052611,GSM1275867
SRR1039516,control,N080611,GSM1275870
SRR1039517,treated,N080611,GSM1275871
SRR1039520,control,N061011,GSM1275874
SRR1039521,treated,N061011,GSM1275875


I think none will be zero in all 8 samples

In [14]:
is_zero = counts == 0
is_zero.head()

,SRR1039508,SRR1039509,SRR1039512,SRR1039513,SRR1039516,SRR1039517,SRR1039520,SRR1039521
ensgene,,,,,,,,
ENSG00000000003,False,False,False,False,False,False,False,False
ENSG00000000005,True,True,True,True,True,True,True,True
ENSG00000000419,False,False,False,False,False,False,False,False
ENSG00000000457,False,False,False,False,False,False,False,False
ENSG00000000460,False,False,False,False,False,False,False,False


In [15]:
all_zero = is_zero.all(axis=1)
all_zero.sum()

np.int64(13436)

13436 are zero in all 8 samples, my prediction was blantantly wrong...